client/src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+
+/**
+ * @class Navbar
+ * @classdesc returns a navbar component 
+ */
+export default class Navbar extends React.Component {
+  render() {
+    return (
+      <nav>
+        <div className="container nav-wrapper">
+          <a href="#!" className="brand-logo">
+            <img src={'https://res.cloudinary.com/djvjxp2am/image/upload/v1509980754/hb-logo_fyj92s.png'} alt="HelloBooks logo"/></a>
```

Comment:
Since it's a string I don't think you really need to use the curly braces. After all you're not concatenating anything

Reply:
Ok sir!